### PR TITLE
Adds support for AR Orientation Tracking

### PR DESCRIPTION
**Pull request by Pilot-Marc**
### Change the ARSession tracking mechanism

Change the way ARSession tracks camera orientation. Before this change ARKit was configured to use real world surfaces to orient itself.  Now it just uses the device's accelerometer.  Result is smoother panning due to the simpler tracking mechanism, and more importantly ARKit can "find itself" after a pause/resume event.  

I also added the App Store icon to remove a compiler warning, and fixed an incorrect variable type.

### Breaking Changes
None.

### Meta
N/A

### Checklist

- [x] Appropriate label has been added to this PR (i.e., Bug, Enhancement, etc.).
- [x] Documentation has been added to all `open`, and `public` scoped methods and properties.
- [x] Changelog has been updated
- [ ] Tests have have been added to all new features. (not a requirement, but helpful)
- [ ] Image/GIFs have been added for all UI related changed.

### Screenshots

### CODE REVIEW

**Comment by Pilot-Marc** [comment]
Change the ARSession tracking mechanism

File: Sources/ARKit-CoreLocation/Location Manager/LocationManager.swift [hunk 1]
```diff
@@ -36,7 +36,7 @@ public class LocationManager: NSObject {
     var currentLocation: CLLocation?
 
     private(set) public var heading: CLLocationDirection?
-    private(set) public var headingAccuracy: CLLocationDegrees?
+    private(set) public var headingAccuracy: CLLocationDirection?
```
Comment (intere) [inline Sources/ARKit-CoreLocation/Location Manager/LocationManager.swift]: 👍 great catch

File: Sources/ARKit-CoreLocation/SceneLocationView.swift [hunk 2]
```diff
@@ -138,8 +138,7 @@ public extension SceneLocationView {
 
     func run() {
         // Create a session configuration
-        let configuration = ARWorldTrackingConfiguration()
-        configuration.planeDetection = .horizontal
+        let configuration = AROrientationTrackingConfiguration()
```
Comment (intere) [inline Sources/ARKit-CoreLocation/SceneLocationView.swift]: `ARWorldTrackingConfiguration`: A configuration that monitors the iOS device’s position and orientation, while enabling you to augment the environment that’s in front of the user.
`AROrientationTrackingConfiguration` - A configuration you use when you just want to track the device’s orientation using the device’s back camera.

Great catch, I am going to need to run some tests on this, but at a glance, this seems perfectly reasonable.

Comment (Pilot-Marc) [reply Sources/ARKit-CoreLocation/SceneLocationView.swift]: In our usage case, we get input from CLLocation and CLHeading for the 3DOF.  While 6DOF would be better, there's no plane in the real world to sync to, and in fact if the OS did locate and sync to a plane, that'd would be destructive.

**Comment by intere** [comment]
@Pilot-Marc and I collaborated on this and we added the ability to use the default AR configuration (World Tracking, without any code changes) - or specify Device Orientation Tracking as a parameter you pass into the `SceneLocationView` initializer.

**Pull request merged**